src/locale/locales/de/messages.po

```diff
@@ -762,23 +724,23 @@ msgstr "Community-Richtlinien"
 
 #: src/screens/Onboarding/StepFinished.tsx:148
 msgid "Complete onboarding and start using your account"
-msgstr ""
+msgstr "Schließen Sie das Onboarding ab und nutzen Sie Ihr Konto"
 
 #: src/view/com/auth/create/Step3.tsx:73
 msgid "Complete the challenge"
-msgstr ""
+msgstr "Beende die Herausforderung"
 
 #: src/view/com/composer/Composer.tsx:417
 msgid "Compose posts up to {MAX_GRAPHEME_LENGTH} characters in length"
-msgstr ""
+msgstr "Verfassen von Beiträgen mit einer Länge von bis zu {MAX_GRAPHEME_LENGTH} Zeichen"
```

Comment:
Strictly speaking, your translation starts with "composition of posts",

maybe

```suggestion
msgstr "Verfasse Beiträge mit einer Länge von bis zu {MAX_GRAPHEME_LENGTH} Zeichen"
```

Reply:
Great, thanks 👍